src/components/ArchivePage/ArchivePageTemplate.js

```diff
@@ -115,10 +74,27 @@ function ArchivePageTemplate() {
 			</div>
 		</div>);
 
-	return <Container maxWidth="md" className={classes.container}>
-		<PageTitle align='center'>Workshop Archive</PageTitle>
-		{quarterEvents}
-	</Container>;
+	return <HeadFooter>
+		<SEO title="Workshop Archive" />
+		<Container maxWidth="md" className={classes.container}>
+			<PageTitle align='center'>Workshop Archive</PageTitle>
+			<details className={classes.tagContainer}>
+				<summary><Typography display='inline' >Filter by tag...</Typography></summary>
+				<div className={classes.tagDetails}>
+					<TagList tags={allTags} />
+				</div>
+			</details>
+			{allEvents}
+		</Container>
+	</HeadFooter>;
 }
 
+ArchivePageTemplate.propTypes = {
+	pageContext: PropTypes.shape({
+		sortedQuarters: PropTypes.array.isRequired,
+		quarterEvents: PropTypes.instanceOf(Map).isRequired,
```

Comment:
Just to be sure, is this right? gatsby-node.js has:

```js
	const quarterEvents = Object.fromEntries(quarterEventsMap);
	createPage({
		path: `/archive`,
		component: ArchivePageTemplate,
		context: {
			quarterEvents,
			allTags
		}
	});
```

which would seem to indicate that quarterEvents is a normal object, not a Map object.

This would look more correct:

```js
quarterEvents: PropTypes.objectOf(PropTypes.array.isRequired).isRequired,
```

Reply:
Good catch, forgot to change all the proptype validations